test/src/test/java/hudson/cli/UpdateViewCommandTest.java

```diff
@@ -66,6 +68,38 @@
         assertThat(result.stderr(), containsString("ERROR: user is missing the View/Configure permission"));
     }
 
+    /**
+     * This test shows that updating a view using an XML that will be
+     * converted by XStream via an alias will rightfully succeed.
+     *
+     * @throws Exception
```

Comment:
Or avoid javadoc (`/*`) for tests, it is not generated anyway. `@throws Exception` on tests is quite pointless.